getstarted.md

```diff
@@ -19,57 +19,47 @@ XXX pending [react-hot-boilerplate/pull/61](https://github.com/gaearon/react-hot
 
 ### Step 1/3: Enabling Hot Module Replacement (HMR)
 
-HMR allows us to replace modules in-place without restarting the server, here's how you can enable it:
+HMR allows us to replace modules in-place without restarting the server, here's how you can enable it for different bundlers:
 
 #### Webpack
 
-* Create a development Webpack config separate from production one
-* Add HotModuleReplacementPlugin to development Webpack config
-* If you only render on the client, consider using WebpackDevServer
-  * Easier to set up
-  * Enable hot: true and add its entry points
-* If you use server rendering, consider using Express server + webpack-dev-middleware
-* More work but also more control
-* Show how to add webpack-dev-middleware and its entry point
+**Option 1: WebpackDevServer (client only)**
 
-**XXX cleanup, details**
+If you only render on the client, this is the easier option and quicker to setup.  The key aspect of this configuration is that when creating a `new WebpackDevServer`, you need to specify `hot: true` as an option. For example, you can add an entirely new file called `server.js` and simply include the [server provided in the boilerplate](https://github.com/gaearon/react-hot-boilerplate/blob/master/server.js){:target="_blank"}.
```

Comment:
> For example, you can add an entirely new file called `server.js` and simply include the server provided in the boilerplate.

At first, I didn't realize you meant to just copy the boilerplate's `server.js` into my project.  Maybe we could rewrite this paragraph to:

> If you're only rendering on the client side, this is the easier option, and quicker to setup.  You can simply copy the `server.js` from the boilerplate into your project. The key aspect of this configuration is that when creating a `new WebpackDevServer`, you need to specify `hot: true` as an option.

and then maybe just include the `server.js` code as a codeblock.  What do you think?